src/sql/workbench/services/connection/browser/cmsConnectionWidget.ts

```diff
@@ -95,13 +95,12 @@ export class CmsConnectionWidget extends ConnectionWidget {
 		super.addAuthenticationTypeOption();
```

Comment:
This should be passing authTypeChanged down - yes it technically doesn't matter at this point but if we changed it so the base class DID care we'd be in trouble then

Reply:
authTypeChanged is just for CMS, and hence the new class for it. This is a shortcoming of the connectionDialog, I tried to generalize as much as possible. If I went to clean everything, we will end up refactoring the whole thing like we spoke about earlier. I had a different way earlier, but making into a new class was the cleanest way.

Reply:
If you want any other changes, I suggest opening a new issue for the connection dialog refactor as that will take some good amount of time and planning.

Reply:
I don't really think this is worth a separate issue (especially since it currently doesn't actually have any impact). But if you absolutely don't want to fix it here then I'll open one.

Reply:
Oh I meant all the "hacks" for CMS in general, not this one in particular. I'll make the asked changes here.